src/electron/electron/core.cljs

```diff
@@ -228,7 +228,42 @@
                                   (if mac?
                                     {:role "close"}
                                     {:role "quit"})]}
-                       {:role "editMenu"}
+                       {:role "editMenu"
+                        ;; https://github.com/electron/electron/blob/85f41d59aceabbdeee1fdec75770249c6335e73a/lib/browser/api/menu-item-roles.ts#L239-L276
+                        :submenu (concat
+                                  [{:label "Undo"
+                                    :click (fn []
+                                             (let [browser-window ^js/BrowserWindow @*win
+                                                   web-contents (.-webContents browser-window)]
+                                               (.send web-contents "invokeEditorHandler" "undo")))
+                                    :accelerator "CommandOrControl+Z"}
```

Comment:
These shortcuts might be conflicts with `src/main/frontend/modules/shortcut/config.cljs`.
I'll check this.

Reply:
From my simple testing, the main process of electron seems only handle these shortcuts(`:accelerator`) and `:click` function  when menu Edit is opened.